### DISCUSSION TITLE
电脑访问主页,左下角显示公网ip地址

### DISCUSSION THREAD

**Discussion by KosLzz** [AlistGo/alist#1992] [Ideas]
**电脑访问主页,鼠标停留文件时,左下角显示公网ip搭建的服务器地址,安全性不足**
[https://pic1.imgdb.cn/item/6346f64c16f2c2beb17490f2.png](url)

**Comment by orzonzofz** [comment]
那你就绑个域名啊

**Comment by anwen-anyi** [comment]
=，= 换个浏览器就行 谷歌会显示360安全不会显示..

**Reply by orzonzofz** [reply]
如果没有绑定域名，纯ip访问的话，edge应该也是显示ip的

**Reply by 12189108** [reply]
掩耳盗铃的做法，不如绑定域名套cdn
